earthpy/__init__.py

```diff
@@ -10,3 +10,11 @@
 epsg = json.loads(
     resource_string("earthpy", "example-data/epsg.json").decode("utf-8")
 )
+
+
+def load_epsg():
+    """ A function to return a dictionary of EPSG code to Proj4 string mappings."""
+
+    return json.loads(
+        resource_string("earthpy", "example-data/epsg.json").decode("utf-8")
+    )
```

Comment:
SO i think AFTER  the function you can define epsg like this:

`epsg = load_epsg()`

And by doing this the existing behavior is mimicked. 

FINALLY -- you will need a test to ensure epsg works. I think what you could do is just call `epsg["some-code-here"]` and assert that the return is the proj4 that you expect.